starport/interface/cli/starport/cmd/network_proposal_approve.go

```diff
@@ -43,14 +48,48 @@ func networkProposalApproveHandler(cmd *cobra.Command, args []string) error {
 
 	var reviewals []spn.Reviewal
 
+	// Get the list of proposal ids
 	ids, err := numbers.ParseList(proposalList)
 	if err != nil {
 		return err
 	}
+
+	// Verify the proposals are valid
+	noVerification, err := cmd.Flags().GetBool(flagNoVerification)
+	if err != nil {
+		return err
+	}
+	if noVerification {
+		// If no verification, ask for a confirmation from the user
+		prompt := promptui.Prompt{
+			Label: fmt.Sprintf("You may approve incorrect proposals. Do you want to continue?"),
```

Comment:
Open for other suggestion

Reply:
I think having a flag is enough, don't need a prompt for this.

Reply:
That's fine for me as well to not include it